src/Microsoft.PowerShell.Commands.Management/commands/management/TestConnectionCommand.cs

```diff
@@ -64,6 +65,12 @@ public class TestConnectionCommand : PSCmdlet, IDisposable
 
         #endregion
 
+        #region Cancellation support
+
+        private readonly CancellationTokenSource cancellationTokenSource = new CancellationTokenSource();
```

Comment:
Our code conventions:
```suggestion
        private readonly CancellationTokenSource _cancellationTokenSource = new CancellationTokenSource();
```

Reply:
Sorted!